pkg/generated-assets/generated_assets.go

```diff
@@ -275,3 +275,22 @@ func getYAMLKind(yaml []byte) (string, error) {
 	}
 	return gvk.GroupKind().String(), nil
 }
+
+func sortAssets(names []string) []string {
+	var stage1, stage2, stage3 []string
+	for i := range names {
+		switch {
+		case strings.HasSuffix(names[i], "_sa.yaml") || strings.HasSuffix(names[i], "_role.yaml"):
```

Comment:
I think service accounts should be created as the last, after all roles and rolebindings are created. A missing SA will produce a different error message than "unable to validate against any security context constraint" and the test will more likely pass ;-)